ephemeris/common_parser.py

```diff
@@ -3,11 +3,17 @@
 import argparse
 
 
-def get_common_args(login_required=True):
+def get_common_args(login_required=True, log_file = False):
 
     parser = argparse.ArgumentParser(add_help=False)
-    general_group = parser.add_argument_group('Galaxy connection')
+    general_group = parser.add_argument_group('General options')
```

Comment:
The general group had the same name as the connection group. This is now also fixed.